CHANGELOG.md

```diff
@@ -7,6 +7,83 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [0.10.0] - 2024-03-03
+
+### Overview
+
+#### Introducing multi-factor authentication
+
+With this release, we are introducing MultiFactorAuthentication and TOTP, this will let you:
+
+-   require (2FA or MFA) during sign in
+-   make use of our TOTP
+
+Check our [guide](https://supertokens.com/docs/thirdpartyemailpassword/common-customizations/multi-factor-auth/overview) for more information.
+
+To use this you'll need compatible versions:
+
+-   Core>=8.0.0
+-   supertokens-node>=17.0.0
+-   supertokens-website>=17.0.3
+-   supertokens-web-js>=0.9.0
+-   supertokens-auth-react>=0.36.0
```

Comment:
this is not necessary, cause it's backwards compatible. So, you should move this into the changes part, under the new recipe and say that in order to use MFA or TOTP, you will need to upgrade the versions of the core and backend SDK.